core/src/main/scala/org/http4s/headers/X-B3-Flags.scala

```diff
@@ -0,0 +1,59 @@
+package org.http4s
+package headers
+
+import org.http4s.parser.HttpHeaderParser
+import org.http4s.util.Writer
+import org.http4s.{ Header, HeaderKey, ParseResult }
+
+import scalaz.ISet
+
+object `X-B3-Flags` extends HeaderKey.Internal[`X-B3-Flags`] with HeaderKey.Singleton {
+  override def parse(s: String): ParseResult[`X-B3-Flags`] =
+    HttpHeaderParser.X_B3_FLAGS(s)
+
+  sealed trait Flag extends Product with Serializable {
+    def longValue: Long
+  }
+
+  object Flag {
+    case object Debug extends Flag {
+      override def longValue: Long = 1 << 0
```

Comment:
Thank you for preserving the symmetry with the other flags with the 0 bit-shift. 👍